src/os_db.mli

```diff
@@ -68,62 +90,112 @@ module User : sig
     ?validity:int64 ->
     act_key:string -> userid:int64 -> email:string -> unit -> unit Lwt.t
 
+  (** [add_preregister email] preregisteres [email] in the database. *)
   val add_preregister : string -> unit Lwt.t
 
+  (** [remove_preregister email] removes [email] from the database. *)
   val remove_preregister : string -> unit Lwt.t
 
+  (** [is_preregistered email] returns [true] if [email] is already registered.
+   * *)
   val is_preregistered : string -> bool Lwt.t
 
+  (** [all ~limit ()] get all email addresses with a limit of [limit]. *)
   val all : ?limit:int64 -> unit -> string list Lwt.t
 
+  (** [create ~password ~avatar ~firstname ~lastname email] creates a new user
+      in the database and returns the userid of the new user.
+      Email is mandatory to create a new user.
+   *)
   val create :
     ?password:string ->
     ?avatar:string ->
     firstname:string -> lastname:string -> string -> int64 Lwt.t
 
+  (** [update ~password ~avatar ~firstname ~lastname userid] updates the user
+      profile with [userid]. *)
   val update :
     ?password:string ->
     ?avatar:string ->
     firstname:string -> lastname:string -> int64 -> unit Lwt.t
 
+  (** [update_password new_password userid] updates the password of the user
+      with ID [userid]. *)
   val update_password : string -> int64 -> unit Lwt.t
 
+  (** [update_avatar new_avatar userid] updates the avatar of the user
+      with ID [userid]. *)
   val update_avatar : string -> int64 -> unit Lwt.t
 
+  (** [update_password new_password userid] updates the password of the user
+      with ID [userid]. *)
   val update_main_email : userid:int64 -> email:string -> unit Lwt.t
 
+  (** [verify_password ~email ~password] verifies if user with email [email] is
+      registered with the password [password] *)
   val verify_password : email:string -> password:string -> int64 Lwt.t
 
+  (** [user_of_userid userid] returns a tuple [(userid, firstname, lastname,
+      main_email, password, avatar)] describing the information about the user.
+   *)
   val user_of_userid :
     int64 -> (int64 * string * string * string option * bool) Lwt.t
 
+  (** [get_activationkey_info key] get the information about the activationkey
```

Comment:
You've used a lot of "returns"... why don't you use "gets" (instead of "get")? Consistency! ;-)